mlflow/server/handlers.py

```diff
@@ -261,12 +262,62 @@ def initialize_backend_stores(backend_store_uri=None, default_artifact_root=None
     except UnsupportedModelRegistryStoreURIException:
         pass
 
+def _assert_string(x):
+    assert isinstance(x, str)
+
+def _assert_int(x):
+    assert isinstance(x, int)
```

Comment:
Can we also include the case where `x` is an int-ish string for consistency with protobuf parsing?

Reply:
Do you want to include this in a separate method? Or just change this (and the float one) to be `_assert_intlike` and try something like this?:

```python
def _assert_int(x):
    try:
        x = int(x)
    except ValueError:
        pass
        
    assert isinstance(x, int)
```

Reply:
I think this is great!